test/bin/mdx-test/expect/envs/test-case.md

```diff
@@ -60,7 +60,8 @@ type t2 = t1
 type t2 = t1
 ```
 
-```ocaml version<4.08,env=e2
+<!-- $MDX version<4.08 -->
```

Comment:
Can you leave this test untouched and add a new test that ensure the new labels are supported and properly printed back?

Reply:
done